config/manager/modeljob-operator.yaml

```diff
@@ -27,17 +27,17 @@ spec:
         - /modeljob-operator
         args:
         - --enable-leader-election
-        image: cleveross/model-registry-controller:latest
-        name: model-registry-controller
+        image: harbor.caicloud.com/release/modeljob-operator:4066990-dirty
+        name: modeljob-operator
         env:
-        - name: HARBOR_DOMAIN
-          value: "test.harbor.com"
-        - name: HARBOR_USERNAME
+        - name: ORMB_DOMAIN
+          value: "harbor.caicloud.com"
```

Comment:
Why do we need the environment variable since we have `   model: "harbor.caicloud.com/release/savedmodel:v1"`?

Reply:
the environment is preset domain, it is match with following ORMB_USERNAME and ORMB_PASSWORD, maybe the domain in model is other domain which is diffenrent from preset domain in the furture. for example: CustomSource. 

now it looks like useless, even unnecessary.

Reply:
Yeah. I think we can reuse the domain in the `model` field to login.

Reply:
BTW, can we reuse ormb-storage-initializer to pull model?

Reply:
If we reuse ormb-storage-initializer, we can't get status for each stage, We can't get failed reason(unless update ormb-storage-initializer for this case).

Reply:
We can open an issue to keep track of it. Personally, prefer reusing if we can. Currently LGTM

Reply:
OK